canonical-websites/www.ubuntu.com
# Updated the /support/plans-and-pricing page

## Done

- Updated the /support/plans-and-pricing page with updates from product mgmt and mkting
- Cleaned up the layout of tables to be more consistent

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/pricing](http://0.0.0.0:8001/pricing)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- compare to the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.9q52wbh3nkil)


## Issue / Card

Fixes #4748
